src/MinterGateway.sol

```diff
@@ -881,12 +877,13 @@ contract MinterGateway is IMinterGateway, ContinuousIndexing, ERC712Extended {
      * @param newTimestamp_ The timestamp of the collateral update.
      */
     function _updateCollateral(address minter_, uint240 amount_, uint40 newTimestamp_) internal {
-        MinterState storage minterState_ = _minterStates[minter_];
-
-        uint40 lastUpdateTimestamp_ = minterState_.updateTimestamp;
+        uint40 minimumTimestamp_ = UIntMath.max40(
```

Comment:
```suggestion
        uint40 miximumTimestamp_ = UIntMath.max40(
```

Reply:
It actually is the "minimum acceptable timestamp for freshness", but I renamed it to `earliestAllowedTimestamp_` since it is clear it was misunderstood.